### DISCUSSION TITLE
Mouse cursor should hide during gameplay in gamemodes that do not use the mouse at all

### DISCUSSION THREAD

**Discussion by OpenSauce04** [ppy/osu#22584] [Ideas] [answered]
When playing Taiko, sometimes I will accidentally touch my mouse's cable and the mouse will move a small amount, and this causes the cursor to appear on screen somewhere and then disappear shortly afterwards.
Given that the mouse cursor doesn't even do anything in this mode, shouldn't the mouse be hidden entirely unless gameplay is paused?

**Comment by ItsJustSomeDude** [comment]
What about for clicking on the pause/close icon in the bottom right?

**Reply by OpenSauce04** [reply]
> What about for clicking on the pause/close icon in the bottom right?

To be completely honest I just assumed that was a mobile only thing because of how unintuitive it would be on a mouse instead of just using the esc key.
Yeah, you have a good point

**Comment by mk56-spn** [comment]
This feels pretty pointless when you could just move your mouse over 5 cm on your desk.. especially since if implemented the pause button will be unusable as mentioned in a comment by another user. Which will probably lead to a discussion being made a week after that asking for it to be a toggle, which probably isn't something Lazer needs more of, especially for stuff like this ..

**Reply by mk56-spn** [reply]
I was saying that Lazer probably doesn't need more toggles. I never stated anything about you being wrong for opening discussions

**Comment by bdach** [comment, marked as answer]
I think there's a possible compromise here in adding a debounce to the showing of the menu cursor so that it doesn't show on small movements. This is a pretty common complaint from users. I wouldn't dismiss it right away.

**Reply by peppy** [reply]
I'm pretty sure stable does something like this, based on movement distance and inactivity time.
